pkg/slo-controller/noderesource/plugins/gpudeviceresource/plugin.go

```diff
@@ -185,7 +188,12 @@ func (p *Plugin) calculate(node *corev1.Node, device *schedulingv1alpha1.Device)
 		totalKoordGPU.Add(d.Resources[extension.ResourceGPUCore])
 	}
 
-	gpuResources[extension.ResourceGPU] = *totalKoordGPU
+	gpuVendor := device.Labels[extension.LabelGPUVendor]
```

Comment:
Some comments are needed here, such as how huawei.com/npu-core on the Node comes from, and why the controller does not need to report

Reply:
The `huawei.com/npu-core` resource is still reported by controller, though Huawei NPU device plugin would also report it, we keep it for potential future impls without device plugins, and there's no need to introduce special logic for it. The count reported by controller and dp is the same so there's no conflict.

P.S. The only special case for Huawei NPU is that the controller would not report `koordinator.sh/gpu` resource, which has been commented.